src/sundials/sundials_profiler.c

```diff
@@ -523,7 +522,8 @@ SUNErrCode sunCollectTimers(SUNProfiler p)
 
 /* Print out the: timer name, percentage of exec time (based on the max),
    max across ranks, average across ranks, and the timer counter. */
-void sunPrintTimers(int idx, SUNHashMapKeyValue kv, FILE* fp, void* pvoid)
+void sunPrintTimers(SUNDIALS_MAYBE_UNUSED int idx, SUNHashMapKeyValue kv,
```

Comment:
As this is an internal function and `idx` is in fact unused, why not just remove it from the function signature instead?